### DISCUSSION TITLE
Exception when starting lnd on Windows

### DISCUSSION THREAD

**Discussion by teslasystemsx** [lightningnetwork/lnd#9255] [Q&A]
Hello.
I'm trying to start lnd with --lnddir parameter on a Windows machine from a command line, but there is an exception. What's wrong with it?

<img width="426" alt="Exception" src="https://github.com/user-attachments/assets/55cfdfa6-d50e-464e-b356-e74822b92b3e">

**Comment by guggero** [comment]
Did you download the binary for the correct architecture? For example `arm` vs. `amd64`? You'll very likely want the `amd64` variant, unless you run some exotic hardware.

**Reply by teslasystemsx** [reply]
> Did you download the binary for the correct architecture? For example `arm` vs. `amd64`? You'll very likely want the `amd64` variant, unless you run some exotic hardware.

Of course. It's amd64.

**Reply by guggero** [reply]
Seems like Windows 7 is no longer supported: https://github.com/golang/go/issues/64622

You can try building with an older Golang version.
